### DISCUSSION TITLE
[@types/node] pipeline signature incompatible with Node 16 and Node 18

### DISCUSSION THREAD

**Discussion by wyuenho** [DefinitelyTyped/DefinitelyTyped#66039] [Issues with a @types package]
[PipelineSourceFunction](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/stream.d.ts#L1232C27-L1232C27), PipelineTransform and PipelineDestinationIterableFunction and PipelineDestinationPromiseFunction have since gained a `{ signal }` parameter since [Node 16](https://github.com/nodejs/node/blob/v16.20.1/lib/internal/streams/pipeline.js#L252-L264), and since Node 18, each [stream can also be a Web Stream](https://github.com/nodejs/node/blob/v18.16.1/lib/internal/streams/pipeline.js#L380).

[a comment by an automated account is not shown]

**Comment by piotr-cz** [comment]
There is a new related discussion here: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/71919
